releases/release-1.16/release-notes-draft.md

```diff
@@ -18,143 +37,138 @@
 
-  Courtesy of SIG Apps, SIG Cluster Lifecycle, SIG Instrumentation, SIG Node, SIG Scheduling, and SIG Testing
+  Courtesy of SIG Release
 - hyperkube: the `--make-symlinks` flag, deprecated in v1.14, has been removed. ([#80017](https://github.com/kubernetes/kubernetes/pull/80017), [@Pothulapati](https://github.com/Pothulapati))
 
   Courtesy of SIG Release
-- Node labels `beta.kubernetes.io/metadata-proxy-ready`, `beta.kubernetes.io/metadata-proxy-ready` and `beta.kubernetes.io/kube-proxy-ds-ready` are no longer added on new nodes.
-  - ip-mask-agent addon starts to use the label `node.kubernetes.io/masq-agent-ds-ready` instead of `beta.kubernetes.io/masq-agent-ds-ready` as its node selector.
-  - kube-proxy addon starts to use the label `node.kubernetes.io/kube-proxy-ds-ready` instead of `beta.kubernetes.io/kube-proxy-ds-ready` as its node selector.
-  - metadata-proxy addon starts to use the label `cloud.google.com/metadata-proxy-ready` instead of `beta.kubernetes.io/metadata-proxy-ready` as its node selector.
-  - Kubelet removes the ability to set `kubernetes.io` or `k8s.io` labels via --node-labels other than the [specifically allowed labels/prefixes](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/0000-20170814-bounding-self-labeling-kubelets.md&#35;proposal). ([#79305](https://github.com/kubernetes/kubernetes/pull/79305), [@paivagustavo](https://github.com/paivagustavo))
-
-  Courtesy of SIG Auth, SIG Cluster Lifecycle, and SIG Node
 
 
 ## New Features
 
-- kubeadm ClusterConfiguration now supports featureGates: IPv6DualStack: true ([#80145](https://github.com/kubernetes/kubernetes/pull/80145), [@Arvinderpal](https://github.com/Arvinderpal))
-
-  Courtesy of SIG Cluster Lifecycle
-- `kubectl get -w` now takes an `--output-watch-events` flag to indicate the event type (ADDED, MODIFIED, DELETED) ([#72416](https://github.com/kubernetes/kubernetes/pull/72416), [@liggitt](https://github.com/liggitt))
-
-  Courtesy of SIG CLI, and SIG Testing
-- Implement "post-filter" extension point for scheduling framework ([#78097](https://github.com/kubernetes/kubernetes/pull/78097), [@draveness](https://github.com/draveness))
-
-  Courtesy of SIG Scheduling, and SIG Testing
 - Add Bind extension point to the scheduling framework. ([#79313](https://github.com/kubernetes/kubernetes/pull/79313), [@chenchun](https://github.com/chenchun))
 
   Courtesy of SIG Scheduling, and SIG Testing
-- Enable cadvisor ProcessMetrics collecting. ([#79002](https://github.com/kubernetes/kubernetes/pull/79002), [@jiayingz](https://github.com/jiayingz))
+- kubeadm: enable secure serving for the kube-scheduler ([#80951](https://github.com/kubernetes/kubernetes/pull/80951), [@neolit123](https://github.com/neolit123))
 
-  Courtesy of SIG Node
-- Add Pre-filter extension point to the scheduling framework. ([#78005](https://github.com/kubernetes/kubernetes/pull/78005), [@ahg-g](https://github.com/ahg-g))
+  Courtesy of SIG Cluster Lifecycle
+- goos: linux
```

Comment:
@onyiny-ang @jeefy @saschagrunert -- Can we investigate this?

A separate code block in the description for https://github.com/kubernetes/kubernetes/pull/79933 was pulled in here, even though the release note was set to `NONE`.

Reply:
@justaugustus - I'll have a look. There is also another one in there that should have been excluded - `"NONE" (#79392, @Bowenislandsong)`. I'm sure we fixed the parser for this. Need to follow up on that too.

Reply:
The `"NONE" (#79392, @Bowenislandsong)` note would have been excluded with `release-note` tool changes waiting to be merged in [PR kubernetes/release #836](https://github.com/kubernetes/release/pull/836). 

I'll remove it manually from this draft for now.

Reply:
@paulbouwer -- I've just reviewed https://github.com/kubernetes/release/pull/836, can you fix that one up and then we can approve? From there, you can just rerun the tool instead of doing the work manually.

Reply:
The [kubernetes/kubernetes#79933](https://github.com/kubernetes/kubernetes/pull/79933) PR from the comment above, has `release-notes` instead of `release-note` as the block identifier.

I'm not sure it is the right approach to add another special case to the parser. Should we ask the author to edit their PR message? Once @onyiny-ang's [Release Notes Guide PR](https://github.com/kubernetes/community/blob/39b777fe437c8bd1f77dfd2cd3df24b76bb400a1/contributors/guide/release-notes.md) is merged, there should be very clear guidance.

I have removed it manually for this draft of the release notes.

Reply:
Can you link the PR here, instead of the commit?

Reply:
Release Notes Guide PR [kubernetes/community #4004](https://github.com/kubernetes/community/pull/4004)

Reply:
I manually removed the wrong entry from the docs draft: https://docs.google.com/document/d/1eumlrzKtf6mzZ-e65pSKTPmYjp-NDuGMKI2kG3W2tZQ/edit